orca/templates/teleport-proxy/resources/usr/local/bin/teleport_init.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# USE THIS SCRIPT ON SINGLE SERVER CONFIGURATION FOR TESTING PURPOSES ONLY!
+
+
+function gen_github(){
+	# Configure GitHub authentication
+	# GitHub credentials defined in configs/<env>-github.yml
+	sed -e "s/__GITHUB_ID__/$GITHUB_ID/" -e "s/__GITHUB_SECRET__/$GITHUB_SECRET/" /root/github.yaml.tpl > /root/github.yaml
+	tctl create -f /root/github.yaml
+}
+
+function gen_token(){
```

Comment:
If we name it generate_agent_token we won't need to add a comment explaining what it does. :)